apps/studio/prisma/scripts/createSite.ts

```
@@ -0,0 +1,236 @@
+          search: undefined,
+          isGovernment: true,
+        }),
+      })
+      .onConflict((oc) =>
+        oc
+          .column("name")
+          .doUpdateSet((eb) => ({ name: eb.ref("excluded.name") })),
+      )
+      .returning("id")
+      .executeTakeFirstOrThrow()
+
+    await tx
+      .insertInto("Footer")
+      .values({
+        siteId,
+        content: jsonb({
+          contactUsLink: "/contact-us",
+          feedbackFormLink: "https://www.form.gov.sg",
+          privacyStatementLink: "/privacy",
+          termsOfUseLink: "/terms-of-use",
+          siteNavItems: FOOTER_ITEMS,
+        }),
+      })
+      .onConflict((oc) =>
+        oc
+          .column("siteId")
+          .doUpdateSet((eb) => ({ siteId: eb.ref("excluded.siteId") })),
+      )
+      .execute()
+
+    await tx
+      .insertInto("Navbar")
+      .values({
+        siteId,
+        content: jsonb(NAV_BAR_ITEMS),
+      })
+      .onConflict((oc) =>
+        oc
+          .column("siteId")
+          .doUpdateSet((eb) => ({ siteId: eb.ref("excluded.siteId") })),
+      )
+      .execute()
+
+    const { id: blobId } = await tx
+      .insertInto("Blob")
+      .values({ content: jsonb(PAGE_BLOB) })
+      .returning("id")
+      .executeTakeFirstOrThrow()
+
+    await tx
+      .insertInto("Resource")
+      .values({
+        draftBlobId: String(blobId),
+        permalink: "",
+        siteId,
+        type: "RootPage",
+        title: "Home",
+      })
+
+      .onConflict((oc) =>
+        oc.column("draftBlobId").doUpdateSet((eb) => ({
+          draftBlobId: eb.ref("excluded.draftBlobId"),
+        })),
+      )
+      .executeTakeFirstOrThrow()
+
+    console.log(`Added site ${siteName} with id ${siteId} to database`)
+    return siteId
+  })
+
+  await addUsersToSite(
+    siteId,
+    [...ISOMER_ADMINS, ...ISOMER_MIGRATORS].map(
+      (email) => `${email}@open.gov.sg`,
+    ),
+  )
+
+  await updateUserPermissions(siteId, RoleType.Admin)
```

would it be a better idea to not have these functions be included here?

---

actually i want to have them here just because it makes it easier for us to run the script! i put fake values so that it'll fail and provide feedback, which minimises the risk of running a "false" migration (+ it's wrapped in `tx` also)

lmk if you don't think this is sufficient!